Makefile

```diff
@@ -54,6 +54,9 @@ test-before-build:
 	## Check for duplicate links in any particular topic file
 	! git grep url:  _topics/ | sed 's/ \+/ /g' | sort | uniq -d | grep .
 
+	## Check for mistakes typical spell checkers can't catch
+	! git grep -i '[d]iscrete log contract'
```

Comment:
Suggest to append ` | grep .` here to prevent interruption of the build process

Reply:
Oh, we had this problem before, didn't we?  Can you try: `! git --no-pager grep -i '[d]iscrete log contract'`

Reply:
@harding That change works as well (both success and fail behave as expected)